pandas/_libs/tslibs/period.pyx

```diff
@@ -1246,6 +1246,38 @@ cdef class _Period(object):
 
     @property
     def dayofweek(self):
+        """
+        Return the day of the week.
+
+        This attribute returns the day of the week on which the particular
+        starting date for the given period occurs with Monday=0, Sunday=6.
+
+        Returns
+        -------
+        Int
+            Range from 0 to 6(included).
+
+        See also
+        --------
+        Period.dayofyear
+            Return the day of year.
```

Comment:
Can you do this in form of `Period.dayofyear : Return the day of year.` ? (same for one below)